src/test/unit_tests/test_system_servers.js

```diff
@@ -435,6 +436,36 @@ mocha.describe('system_servers', function() {
         });
     });
 
+    mocha.it('Calculate md5_etag for bucket works', async function() {
+        this.timeout(90000); // eslint-disable-line no-invalid-this
+        return P.resolve()
+            .then(() => rpc_client.bucket.create_bucket({
+                name: MD5_BUCKET,
+                force_md5_etag: true,
+            }))
+            .then(() => rpc_client.bucket.read_bucket({
+                name: MD5_BUCKET,
+            }))
+            .then(info => assert(info.force_md5_etag && info.force_md5_etag === true))
```

Comment:
I suggest writing it with await and not using a promise chain. it's much more readable

e.g.:
```suggestion
        await rpc_client.bucket.create_bucket({
             name: MD5_BUCKET,
             force_md5_etag: true,
         });
         const info = await rpc_client.bucket.read_bucket({
                name: MD5_BUCKET,
          })
          assert(info.force_md5_etag && info.force_md5_etag === true);
```

Reply:
Sure, Thanks.